_posts/2023-05-12-sprint-attendees.md

```diff
@@ -0,0 +1,49 @@
+---
+author: DjangoCon Europe Organisers
+category: General
+date: 2023-05-12
```

Comment:
Not super important, but if you're changing other things can you also update the date of the post